### DISCUSSION TITLE
Cloud - navigation logo disappears after update

### DISCUSSION THREAD

**Discussion by MitchOgden** [statamic/cms#4936] [Troubleshooting]
I am using the cloud starter kit on a new site I started designing today. I changed the logo from the default that came with the starter kit and now it no longer displays the logo at the top of the page, only in the footer. Any ideas? No errors showing up in chrome console.

**Comment by edalzell** [comment]
Top of a front end page?

**Comment by jasonvarga** [comment]
I'd ask the question on [their GitHub repo](https://github.com/lucky-media/cloud/discussions). It's not a Statamic product.
